paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/compact/UnawareBucketCompactor.java

```diff
@@ -100,14 +100,41 @@ public void processElement(UnawareAppendCompactionTask task) throws Exception {
                                                         metricsReporter.reportCompactionTime(
                                                                 System.currentTimeMillis()
                                                                         - startMillis);
+                                                        metricsReporter
+                                                                .increaseCompactionsCompletedCount();
                                                     }
                                                 },
                                                 LOG);
                                         return commitMessage;
+                                    } catch (Exception e) {
+                                        MetricUtils.safeCall(
+                                                this::increaseCompactionsFailedCount, LOG);
```

Comment:
when there is a exception, the job will restart and this metric will reset to 0, do we have to count this?

Reply:
haha, actually I don't know and do you have any good suggestions? Thanks!

Reply:
If we do not have a good way to count this, we can remove this metric now.

Reply:
Do you mean that we just remove `CompactionsFailedCount` metric and reserve others metrics `CompactionsFailedCount/CompactionsQueuedCount`？

Reply:
yes, reserve `compactionCompletedCount/compactionQueuedCount`

Reply:
Got it, thanks!